src/lib/Syntax.hs

```diff
@@ -784,11 +798,15 @@ instance BindsUVars UPat' where
 instance HasUVars UDecl where
   freeUVars (ULet _ p expr) = freeUVars p <> freeUVars expr
   freeUVars (UData (UConDef _ bs) dataCons) = freeUVars $ Abs bs dataCons
+  freeUVars (UInterface _ _ _) = mempty  -- TODO
+  freeUVars (UInstance _ _) = mempty     -- TODO
```

Comment:
Seems like something worth fixing before we merge this

Reply:
This was worth doing! Implementing it made me realize I've been sloppy about the scope of the binders in the instance type. They're meant to be in scope in the method implementations, so they shouldn't be represented as alpha-renameable pi binders within the type. I'll push a separate fix.